lib/WooCommerce/Helper.php

```diff
@@ -64,4 +64,8 @@ public function getRawPrice($price, array $args = []) {
     $htmlPrice = $this->wcPrice($price, $args);
     return html_entity_decode(strip_tags($htmlPrice));
   }
+
+  public function getCountries(): array {
+    return (new \WC_Countries)->get_countries() ?? [];
```

Comment:
Good idea. I've discussed this with @NeosinneR and we will go with `get_allowed_countries`. As for `to only display the countries that they actually sold something to`, it would have slower performance + user wouldn't be able to create a segment before they have some orders into the particular country.